reference/constraints/Traverse.rst

```diff
@@ -0,0 +1,94 @@
+Traverse
+========
+
+Objects do not validate nested objects by default unless explicitly using this constraint.
+If only certain nested objects should be validated by cascade, consider using the :doc:`references/constraints/Valid`
+instead.
+
++----------------+-------------------------------------------------------------------------------------+
+| Applies to     | :ref:`class <validation-class-target>`                                              |
++----------------+-------------------------------------------------------------------------------------+
+| Options        | - `payload`_                                                                        |
++----------------+-------------------------------------------------------------------------------------+
+| Class          | :class:`Symfony\\Bridge\\Doctrine\\Validator\\Constraints\\Traverse`                |
++----------------+-------------------------------------------------------------------------------------+
+
+Basic Usage
+-----------
+
+In the following example, create three classes ``Book``, ``Author`` and ``Editor``
+that all have constraints on their properties. Furthermore, ``Book``
+stores an ``Author`` and an ``Editor`` instances that must be valid too::
```

Comment:
I would add one more sentence here to show why you called this the greatest annotation. E.g. "[...] that must be valid too. Instead of adding the `Valid` constraint to both fields, configure the `Traverse` constraint on the `Book` class."

Reply:
Nice, done!